be/src/common/config.h

```diff
@@ -265,6 +265,27 @@ namespace config {
     CONF_mInt64(base_compaction_interval_seconds_since_last_operation, "86400");
     CONF_mInt32(base_compaction_write_mbytes_per_sec, "5");
 
+    // config the cumulative compaction policy
+    // Valid configs: ordinary, universal
```

Comment:
ordinary and universal  naming both meaningless and general, we need a concrete and intuitive naming. You could refer to HBase.